packages/launcher/src/index.tsx

```diff
@@ -339,6 +339,12 @@ export namespace ILauncher {
      * spec.
      */
     kernelIconUrl?: string;
+
+    /**
+     * Metadata about the item.  This can be used by the launcher to
+     * affect how the item is displayed.
+     */
+    metadata?: ReadonlyJSONObject;
```

Comment:
Perhaps the default implementation should display this metadata somewhere? It feels a bit weird to include metadata without using it anywhere. Presumably you have an overridden launcher that actually is using the metadata?

Reply:
Yes, this change was meant to allow experimentation without needing to override core extensions like `notebook` and `console`.

Reply:
Sure, okay.